app/templates/direct-award/view-project.html

```diff
@@ -172,22 +172,15 @@
             ] if project else [])
           },
           {
-            "body": "Complete the Customer Benefits Record form",
+            "body": "Submit a Customer Benefits Record",
             "custom_body_list": [
               {"type": "text", "text":
-                ''.join(["You must fill in the <a href=\"", framework_urls.customer_benefits_record_form_url|e, "\"
+                ''.join(["You must <a href=\"", framework_urls.customer_benefits_record_form_url|e, "\"
                                                  target=\"_blank\" rel=\"external noopener noreferrer\"
                                                  data-analytics=\"trackEvent\"
                                                  data-analytics-category=\"Direct Award\"
                                                  data-analytics-action=\"External Link\"
-                                                >Customer Benefits Record form</a>
-                and email it to the Crown Commercial Service at <a href=\"mailto:", framework_urls.customer_benefits_record_form_email|e, "\"
-                                                                   target=\"_top\"
-                                                                   data-analytics=\"trackEvent\"
-                                                                   data-analytics-category=\"Direct Award\"
-                                                                   data-analytics-action=\"External Link\"
-                                                                  >", framework_urls.customer_benefits_record_form_email|replace("-", "&#8209;"|safe), "</a>.",
-                                                                  "<!-- use non-breaking dash TODO: do this in a better way -->",
+                                                >submit a Customer Benefits Record</a>.",
                  " They will contact you if they find any issues with the supplier."]
```

Comment:
Who is 'They' in `They will contact you`? We've removed the reference to CCS in the previous sentence...

Reply:
Shall we just change it to `CCS will contact you`?

Reply:
Yep. 'The Crown Commercial Service will contact you...'

Reply:
Should Crown Commercial Service be separated by non-breaking spaces? Unfortunately doing that seems to mess things up...

![image](https://user-images.githubusercontent.com/503614/69161043-70d33d00-0ae2-11ea-90f6-ebb28f347e43.png)

Reply:
It wasn't using nbsp before - don't think we do it as a rule for them.